guides/gitconfig.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Managing Git Configuration
+description: Learn how to manage your Git configuration in Coder.
+---
+
+This guide will show you how to manage Git configuration through the system,
+global, and repo levels.
+
+## How to use the System and Global configurations
+
+For customers seeking organizational (system-level) `.gitconfig` defaults, we recommend
```

Comment:
I'd suggest leaving out "For customers". Though again, defer to @khorne3 on final copy

Reply:
```suggestion
Users seeking organizational (system-level) `.gitconfig` defaults, we recommend
```

agreed, would do something like this perhaps